embeddings/bq-vector-search-log-outlier-detection.ipynb

```diff
@@ -0,0 +1,1474 @@
+{
```

Comment:
<p>Make this a bigger header. Maybe h4? <code>####</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/381/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='81'/>

Reply:
<p>The containing header ("Visualize results") is already h4 and I noticed h5 doesn't render in GitHub markdown.....it actually looks like another text which is even less obvious. Hence the intentional html underline...Open to other suggestions for such deeply nested sections.</p>